### DISCUSSION TITLE
Add the 3 Ws to the templates/business-processes/Business Process Catalog October

### DISCUSSION THREAD

**Discussion by SamkhawajaSKH** [microsoft/dynamics365patternspractices#119] [Ideas]
Hi, 
It would be nice to add the following to the templates/business-processes/Business Process Catalog level 4:
When the process is used (Scenarios..)
Who will use the process (user roles /personas = Swim lanes in the graphics)
What to expect from the process (success criteria at time of delivery)

the How to complete the process will be linked to the Graphics... https://github.com/microsoft/dynamics365patternspractices/tree/main/graphics

One more idea to show the personas in the graphics... https://github.com/microsoft/dynamics365patternspractices/tree/main/graphics
Thanks
Sam

**Comment by rachel-profitt** [comment]
Can you please provide some more details about what you are asking for. It is not clear to me if you are asking to have the spreadsheet catalog updated or the learn articles related to a process to have this information. If you intended to include screenshots or graphics, the links provide simply take me to the repository where our source graphics are located.

**Reply by SamkhawajaSKH** [reply]
Hi Rachel,
My suggestion has 2 folds:
1 - For Every level 4, it would be good to define the When, who and what. In other terms possible use cases (business scenarios), and leave a room for being edited during the solution modeling phase of the implementation. The major 2 benefits are: (1) Avoid scope creep (missing a business use case), (2) facilitate the sign off on the User Stories - testing scripts during UAT by defining in advance the case scenarios and the expected results.
2- For the Graphics inter-relation with the L4 sequence, I am attaching an example from the ISD library. 
We can discuss if needed.
Hope that helps!
Sam
[20.45.020 Post Invoice PO, Items.pdf](https://github.com/microsoft/dynamics365patternspractices/files/13547233/20.45.020.Post.Invoice.PO.Items.pdf)
